Week07/lesson14/classwork/responsive_articles_with_menu/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Responsive Articles</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato:300|PT+Serif:700" rel="stylesheet">
+  <link rel="stylesheet" href="css/normalize.css" />
+  <link rel="stylesheet" href="css/styles.css" />
+</head>
+<body>
+  <header>
+    <nav>
+      <article class="clearfix">
+        <menu>
```

Comment:
I'd consider using an `<a>` here if I were you so that you get the little hand when you hover over the hamburger